ceph.spec.in

```diff
@@ -839,7 +839,7 @@ rm -rf %{buildroot}
 %{_sbindir}/ceph-create-keys
 %{_sbindir}/rcceph
 %dir %{_libexecdir}/ceph
-%{_libexecdir}/ceph/ceph_common.sh
+%{_libdir}/ceph/ceph_common.sh
```

Comment:
https://github.com/ceph/ceph/blob/master/ceph.spec.in#L781 uses `_libexecdir`. May I ask why should I delete it?